apps/admin-ui/src/identity-providers/add/AddOpenIdConnect.tsx

```diff
@@ -28,9 +28,8 @@ type DiscoveryIdentity = IdentityProviderRepresentation & {
 export default function AddOpenIdConnect() {
   const { t } = useTranslation("identity-providers");
   const navigate = useNavigate();
-  const { url } = useRouteMatch();
-  const isKeycloak = url.endsWith("keycloak-oidc");
-  const id = `${isKeycloak ? "keycloak-" : ""}oidc`;
+  const { id } = useParams<{ id: string }>();
```

Comment:
Can we re-use the `IdentityProviderOidcParams` type here?
```suggestion
  const { id } = useParams<IdentityProviderOidcParams>();
```